### DISCUSSION TITLE
idle timeout (For when you pause the music)

### DISCUSSION THREAD

**Discussion by Ogwy** [purpl3F0x/TIDAL-Discord-Rich-Presence-UNOFFICIAL#90] [Ideas]
let's say when you pause the music on tidal and start playing a game. Discord shows the tidal status paused, as well as the name and artist, for an ?infinite time?

I think there should be a time out for how long the music is paused. For example: If I pause my music the status will disappear from my discord after 5 mins. Then reappear when i start playing music again. 
hopefully that makes sense

**Comment by purpl3F0x** [comment]
Question for thought, If you have tidal opened and paused a song what is the status of TIDAL? Paused? No?
